pegasus/forms/volunteer_engineer_submission_2015.rb

```diff
@@ -9,8 +9,9 @@ class VolunteerEngineerSubmission2015 < VolunteerEngineerSubmission
   # and after each Hour of Code.
   UNSUBSCRIBE_HOC = "untilhoc".freeze
   UNSUBSCRIBE_FOREVER = "forever".freeze
-  DEFAULT_DISTANCE = 24 # kilometers
-  DEFAULT_NUM_VOLUNTEERS = 10
+  DEFAULT_DISTANCE_MILES = 20
+  DEFAULT_DISTANCE_KM = 32
```

Comment:
How are MILES vs KM used in this system?

Reply:
We use KM value in SQL query, which uses [ST_Distance_Sphere](https://dev.mysql.com/doc/refman/5.7/en/spatial-convenience-functions.html). 
https://github.com/code-dot-org/code-dot-org/blob/bed88da02d544205a802c941b0665319b9005156/pegasus/forms/volunteer_engineer_submission_2015.rb#L187
We use MILES value in UI.